lib/ansible/modules/cloud/scaleway/scaleway_snapshots_facts.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+#
+# Scaleway snapshots facts module
+#
+# Copyright (C) 2018 Online SAS.
+# https://www.scaleway.com
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: scaleway_snapshots_facts
+short_description: Scaleway snapshots facts module
+version_added: "2.7"
+author: Remy Leone (@sieben)
+description:
+    - "This module gather facts about snapshots on Scaleway."
+options:
+
+  oauth_token:
```

Comment:
Other module haved aliased `oauth_token` to `api_token` since `api_token` is what's actually used in the code I think it makes sense to add the alias here.